app/js/main.js

```diff
@@ -346,7 +359,7 @@ function undoFrame() {
     "use strict";
     // Make sure there is a frame to delete
     if (curFrame > 0) {
-      deleteFrame(curFrame);
+      confirmSet("deleteFrame", curFrame, "Are you sure you want to delete the last frame captured?");
```

Comment:
Then with the revised function, this and other calls change to
`confirmSet(deleteFrame, curFrame, "Are you sure you want to delete the last frame captured?");`

The change being the function name is no longer a string.